app/main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "github.com/ngavinsir/golangtraining/app/cmd"
+
+func main() {
+	cmd.Execute()
```

Comment:
cmd.Execute pattern is specific only to `spf13/cobra` library usage, if you don't use the library, you don't have to use this pattern